package.json

```
@@ -48,13 +48,16 @@
     "copy:files": "copyfiles -u 1 -V './src/auroElement/**/*' ./dist",
     "css:lint": "stylelint './results/css/*.css'",
     "scss:lint": "stylelint './src/**/*.scss'",
-    "build": "npm-run-all test build:sassdoc build:sass:demo build:sass:test build:sass:essentials build:sass:essentials+ copy:files postinstall",
+    "build": "npm-run-all test build:sassdoc build:sass:demo build:sass:test build:sass:essentials-ac build:sass:essentials-as build:sass:essentials-ha build:sass:essentials-asc build:sass:essentials+ copy:files postinstall",
```

this might make it a little more readable.

```suggestion
    "build": "npm-run-all test build:sassdoc build:sass:demo build:sass:test build:themes build:sass:essentials+ copy:files postinstall",
    "build:themes": "build:sass:essentials-ac build:sass:essentials-as build:sass:essentials-ha build:sass:essentials-asc",
```